frontend/model/state.js

```diff
@@ -157,6 +157,28 @@ sbp('sbp/selectors/register', {
           upgradeAction(contractID)
         })
       }
+    })();
+    (() => {
+      // Update expired invites
+      // If fewer than MAX_GROUP_MEMBER_COUNT 'anyone can join' have been used,
+      // create a new 'anyone can join' link up to MAX_GROUP_MEMBER_COUNT invites
+      const ourIdentityContractId = state.loggedIn?.identityContractID
+      if (!ourIdentityContractId || !state[ourIdentityContractId]?.groups) return
+      Object.entries(state[ourIdentityContractId].groups).map(([groupID, { hasLeft }]: [string, Object]) => {
+        const groupState = state[groupID]
+        if (hasLeft || !groupState?.invites) return undefined
+        const now = Date.now()
+        const usedInvites = Object.keys(groupState.invites)
+          .filter(invite => groupState.invites[invite].creatorID === INVITE_INITIAL_CREATOR)
+          .reduce((acc, cv) => acc +
+        (!(groupState._vm.invites[cv].expires >= now)
```

Comment:
All anyone-can-join invites need to be updated, regardless of whether or not they're expired.

Reply:
They're not updated, a new invite is created. This logic takes into account how many people to allow using them, as explained in the comments.